.github/workflows/check_docs.yml

```diff
@@ -0,0 +1,55 @@
+name: Docs checks
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+  schedule:
+    # run every Monday at 6am UTC
+    - cron: '0 6 * * 1'
+
+env:
+  SETUP_XVFB: True  # avoid issues if mpl tries to open a GUI window
+  TOXARGS: '-v'
+
+jobs:  # This job is used to check that the documentation builds correctly.
```

Comment:
I'm not sure I've fully grasped what the advantage of separating this out is — it seems like we're replacing one small stanza in the `ci_test.yml` file with a bunch of duplicated boilerplate here. Is the motivation to reduce testing time when you are only working on docs? Or do you have something else in mind?

Reply:
Basically, I *thought* that the `if` line above did something and was not hardcoded in Github (something similar in many example configurations and Stackoverflow answers), but apparently this has stopped working recently. In which case, I could stop the `ci_test` with `[docs only]`, but not the `check_docs` tests. But this does not work, so I guess we can put back the small stanza in ci_test.yml